apps/modernization-ui/src/apps/search/laboratory-report/labReportFormTypes.ts

```diff
@@ -34,7 +36,8 @@ type General = {
     pregnancyStatus?: Selectable;
     processingStatus?: Selectable[];
     programAreas?: Selectable[];
-};
+    providerType?: Selectable;
+} & ReportCriteria;
```

Comment:
nit: This is redundant, the `LabReportFiterEntry` is composed of `General` and `ReportCriteria`.